src/symbol_table.cc

```diff
@@ -80,6 +80,15 @@ auto DonsusSymTable::get(std::string qualified_name) -> sym {
   return b;
 }
 
+auto DonsusSymTable::setInst(std::string qualified_name, llvm::AllocaInst *inst)
+    -> void {
+  for (auto &n : underlying) {
+    if (n.key == create_qualified_name(qualified_name)) {
```

Comment:
I reckon - using references to pointers might be a better solution, exposing such an API could lead to unnecessary additions.

Reply:
well yeah